easybuild/easyconfigs/v/VarScan/VarScan-2.3.6-Java-1.7.0_80.eb

```diff
@@ -13,7 +13,7 @@ description = """Variant calling and somatic mutation/CNV detection for next-gen
 
 toolchain = {'name': 'dummy', 'version': ''}
 
-sources = [('%(name)s.v%(version)s.jar', 'download')]
+sources = ['%(name)s.v%(version)s.jar']
```

Comment:
Good question, I was wondering the exact same thing when I ran into it...

It's a bug, this syntax was used to specify a custom extract command, but since the JAR easyblock just skips the extract step this went unnoticed.